WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift

```diff
@@ -214,13 +212,33 @@ class MeViewController: UITableViewController {
                                               accessibilityIdentifier: "About"))
 
                 return rows
-            }()),
+            }())
+        ]
+
+#if JETPACK
```

Comment:
[Question] Why are we using macros here? Have you considered using `AppConfiguration.isJetpack`?

Reply:
I can answer this question as it was my suggestion @hassaanelgarem.

Without the macro, the `MyDomainsViewController` files and all the files that depend on it would have to be added to WordPress target too. Even though the My Domains screen is for Jetpack only.

And if we remove those files from WordPress target, then the WordPress app won't build.